src/app/backend/resource/container/logs.go

```diff
@@ -100,16 +119,33 @@ func getRawPodLogs(client *client.Clientset, namespace, podID string, logOptions
 
 // Build logs structure for given parameters.
 func ConstructLogs(podID string, rawLogs string, container string, logSelector *logs.Selection) *logs.LogDetails {
-	logLines, fromDate, toDate, logSelection := logs.ToLogLines(rawLogs).SelectLogs(logSelector)
+	parsedLines := logs.ToLogLines(rawLogs)
+	logLines, fromDate, toDate, logSelection, lastPage := parsedLines.SelectLogs(logSelector)
+
+	readLimitReached := isReadLimitReached(int64(len(rawLogs)), int64(len(parsedLines)), logSelector.LogFilePosition)
+	truncated := readLimitReached && lastPage
+
 	info := logs.LogInfo{
 		PodName:       podID,
 		ContainerName: container,
 		FromDate:      fromDate,
 		ToDate:        toDate,
+		Truncated:     truncated,
 	}
 	return &logs.LogDetails{
 		Info:      info,
 		Selection: logSelection,
 		LogLines:  logLines,
 	}
 }
+
+// Checks if the amount of log file returned from the apiserver is equal to the read limits
+func isReadLimitReached(bytesLoaded int64, linesLoaded int64, logFilePosition string) bool {
+	if logFilePosition == logs.Beginning && bytesLoaded >= byteReadLimit {
+		return true
+	}
+	if logFilePosition == logs.End && linesLoaded >= lineReadLimit {
+		return true
+	}
+	return false
```

Comment:
You can do `return (logFilePosition == logs.End && linesLoaded >= lineReadLimit) || (logFilePosition == logs.End && linesLoaded >= lineReadLimit)` instead.